src/__tests__/imagings/requests/NewImagingRequest.test.tsx

```diff
@@ -1,214 +1,148 @@
 import { createMemoryHistory } from 'history'
 import React from 'react'
-import { act } from 'react-dom/test-utils'
 import { Provider } from 'react-redux'
 import { Router, Route } from 'react-router-dom'
+import selectEvent from 'react-select-event'
 import createMockStore from 'redux-mock-store'
 import thunk from 'redux-thunk'
 
 import NewImagingRequest from '../../../imagings/requests/NewImagingRequest'
 import * as breadcrumbUtil from '../../../page-header/breadcrumbs/useAddBreadcrumbs'
 import * as ButtonBarProvider from '../../../page-header/button-toolbar/ButtonBarProvider'
 import * as titleUtil from '../../../page-header/title/TitleContext'
-import SelectWithLabelFormGroup from '../../../shared/components/input/SelectWithLabelFormGroup'
-import TextFieldWithLabelFormGroup from '../../../shared/components/input/TextFieldWithLabelFormGroup'
-import TextInputWithLabelFormGroup from '../../../shared/components/input/TextInputWithLabelFormGroup'
-import ImagingRepository from '../../../shared/db/ImagingRepository'
-import Imaging from '../../../shared/model/Imaging'
-import Patient from '../../../shared/model/Patient'
 import { RootState } from '../../../shared/store'
+import { expectOneConsoleError } from '../../test-utils/console.utils'
 
 const mockStore = createMockStore<RootState, any>([thunk])
 
 describe('New Imaging Request', () => {
   let history: any
   let setButtonToolBarSpy: any
 
-  const setup = async () => {
+  const setup = () => {
     jest.resetAllMocks()
     jest.spyOn(breadcrumbUtil, 'default')
     setButtonToolBarSpy = jest.fn()
-    jest.spyOn(titleUtil, 'useUpdateTitle').mockImplementation(() => jest.fn())
     jest.spyOn(ButtonBarProvider, 'useButtonToolbarSetter').mockReturnValue(setButtonToolBarSpy)
 
     history = createMemoryHistory()
     history.push(`/imaging/new`)
     const store = mockStore({} as any)
 
-    let wrapper: any
-    await act(async () => {
-      wrapper = await mount(
-        <ButtonBarProvider.ButtonBarProvider>
-          <Provider store={store}>
-            <Router history={history}>
-              <Route path="/imaging/new">
-                <titleUtil.TitleProvider>
-                  <NewImagingRequest />
-                </titleUtil.TitleProvider>
-              </Route>
-            </Router>
-          </Provider>
-        </ButtonBarProvider.ButtonBarProvider>,
-      )
-    })
-    wrapper.find(NewImagingRequest).props().updateTitle = jest.fn()
-    wrapper.update()
-    return wrapper as ReactWrapper
+    return render(
+      <ButtonBarProvider.ButtonBarProvider>
+        <Provider store={store}>
+          <Router history={history}>
+            <Route path="/imaging/new">
+              <titleUtil.TitleProvider>
+                <NewImagingRequest />
+              </titleUtil.TitleProvider>
+            </Route>
+          </Router>
+        </Provider>
+      </ButtonBarProvider.ButtonBarProvider>,
+    )
   }
 
-  describe('title and breadcrumbs', () => {
```

Comment:
I see this test was removed. Any particular reasoning why we decided to leave this one out?

Reply:
RTL pattern, files themselves are "suites" 
We kept ones that did isolate logic so as not to re-architect things but this is part of the Enzyme conversion

Reply:
> RTL pattern, files themselves are "suites"
> We kept ones that did isolate logic so as not to re-architect things but this is part of the Enzyme conversion to RTL
> 😄
> 
> RTL Author Blog on Nesting
> https://kentcdodds.com/blog/avoid-nesting-when-youre-testing

Makes perfect sense. I'll checkout the blog.